shop/docker-compose.yaml

```diff
@@ -44,9 +44,19 @@ services:
         # source: ./themes/myTheme # local path to the theme
         # target:  /var/www/html/themes/myTheme # path to be mounted in the container
       - psdata:/var/www/html
+      - etc:/etc
+      - img:/var/www/html/img
 
 networks:
   presta_network:
 volumes:
   psdata:
   dbdata:
+  etc:
+  img:
```

Comment:
Skoro dodajesz tutaj volume na obrazki to po co są one również w innym miejscu w ZIPie

Reply:
To dałem żeby można było sobie łatwo zzipować ten folder gdyby się wrzuciło więcej obrazków